Marlin/src/HAL/STM32F1/SPI.cpp

```diff
@@ -710,6 +710,10 @@ static spi_baud_rate determine_baud_rate(spi_dev *dev, uint32_t freq) {
   return baud_rates[i];
 }
 
-SPIClass SPI(1);
+#if SPI_DEVICE == 1
+  SPIClass SPI(1);
+#else
+  SPIClass SPI(2);
+#endif
```

Comment:
```suggestion
SPIClass SPI(SPI_DEVICE);
```


`spi_pins.h` allows for three SPI devices. The SKR Mini V1.1 (NOT the E3 Mini) defines `ENABLE_SPI3`. Might as well just use the constant directly.

```
#if ENABLED(ENABLE_SPI3)
  #define SPI_DEVICE 3
#elif ENABLED(ENABLE_SPI2)
  #define SPI_DEVICE 2
#else
  #define SPI_DEVICE 1
#endif
```

Reply:
HAHAH, for sure. I'm too concentred in the lcd and the code that just made a stupid if :-p